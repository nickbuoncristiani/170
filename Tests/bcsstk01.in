48
25
1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48
17 47 10 32 33 1 34 46 35 24 25 37 12 48 38 4 13 41 44 22 43 9 16 21 3
8
x 1.06398 x 1.8063 1.55697 x x x x x x x x x x 1.74937 x 1.7554 1.01254 x x x 1.71975 x x x x x x x x x x x x x x x x x x x x x x x x x
1.06398 x 1.87722 1.52342 1.74573 1.29773 1.83831 1.66705 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x 1.87722 x x x x 1.76635 1.09051 x 1.66924 x x 1.08251 1.53786 x x x x x x x x x 1.72849 x x x x x x x x x x x x x x x x x x x x x x x x
1.8063 1.52342 x x 1.51369 x x x x x x x x x x x x x x x x x x 1.45337 1.0983 1.9354 1.48067 x x x x x x x x x x x x x x x x x x x x x
1.55697 1.74573 x 1.51369 x x x x x x x x x x x x 1.75071 x x x x x x x x x x x x x 1.25345 1.08944 x x x 1.4328 x x x x x x x x x x x x
x 1.29773 x x x x x x x x x x x 1.97422 x x x x 1.04962 x x x x x 1.59808 x x x x x x 1.33248 x x x x x x 1.07438 x 1.40389 x x x x x x x
x 1.83831 1.76635 x x x x 1.92117 x x x x x x x x x x x x x x 1.20893 x x 1.65917 x x x x x x x x x 1.01789 x x x x x x x x x x x x
x 1.66705 1.09051 x x x 1.92117 x x x x x x x x x x x x x x x x x x x 1.60727 x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x 1.66571 1.85786 1.41649 x x x 1.93147 x x x 1.14802 1.74181 1.31836 x x x x x x x x x x x x x x x x x x x x x x x x x x
x x 1.66924 x x x x x 1.66571 x 1.66753 1.66616 1.91543 1.04741 1.88245 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x 1.85786 1.66753 x 1.89392 x x x x x x x x x x x 1.54503 x x x 1.22265 1.93824 1.70932 x x x x x x x x x x x x x x x x x x
x x x x x x x x 1.41649 1.66616 1.89392 x x x x x 1.43208 x x x x x x x x x x x x x x x 1.73451 1.38332 1.30693 x x x x x x x x x x x x x
x x 1.08251 x x x x x x 1.91543 x x x 1.63534 x x x x x x x 1.20373 x x x x x 1.46658 x x x x x x 1.78019 x x x x x x x 1.45884 x x x x x
x x 1.53786 x x 1.97422 x x x 1.04741 x x 1.63534 x x x x x x x x x x x x x x x 1.15376 x x x x x x x x x 1.74855 x 1.62743 x x x x x x x
x x x x x x x x x 1.88245 x x x x x x x x x x 1.64811 x x x x x x x x 1.78958 x x x 1.31054 x x x x x x x x x x x x x x
1.74937 x x x x x x x 1.93147 x x x x x x x 1.13615 1.44567 1.17111 1.185 1.71205 x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x 1.75071 x x x x x x 1.43208 x x x 1.13615 x x x x x x x x x x x x x x 1.90993 1.20661 1.2079 1.46994 x x x x x x x x x x x x x x
1.7554 x x x x x x x x x x x x x x 1.44567 x x 1.80154 x x 1.83486 x x x x x x x x 1.76886 x x x x x x x x x x x x 1.75345 1.93443 x x x
1.01254 x x x x 1.04962 x x x x x x x x x 1.17111 x 1.80154 x x x x x x 1.86329 x x x x x x 1.53023 x x x x x x x 1.95878 x x x x x x x x
x x x x x x x x 1.14802 x x x x x x 1.185 x x x x 1.16359 x x x x x x x x x x x 1.45569 x x x x x x x x x x x x x x x
x x x x x x x x 1.74181 x x x x x 1.64811 1.71205 x x x 1.16359 x x x x x x x x x 1.61613 x x x 1.60929 x x x x x x x x x x x x x x
x x x x x x x x 1.31836 x x x 1.20373 x x x x 1.83486 x x x x x x x x x 1.97835 x x x x x x 1.76921 x x x x x x x x 1.31586 1.65585 x x x
1.71975 x x x x x 1.20893 x x x x x x x x x x x x x x x x x x 1.81027 x x x x x x x x x 1.58527 x x x x x x x x x x x x
x x 1.72849 1.45337 x x x x x x 1.54503 x x x x x x x x x x x x x x 1.97322 1.24865 1.4059 1.37146 x x x x x x x x x x x x x x x x x x x
x x x 1.0983 x 1.59808 x x x x x x x x x x x x 1.86329 x x x x x x x x x 1.21006 x x 1.39195 x x x x 1.89186 1.60453 1.47707 1.76987 1.14891 x x x x x x x
x x x 1.9354 x x 1.65917 x x x x x x x x x x x x x x x 1.81027 1.97322 x x 1.41929 x x x x x x x x 1.70246 1.98514 x x x x x x x x x x x
x x x 1.48067 x x x 1.60727 x x x x x x x x x x x x x x x 1.24865 x 1.41929 x x x 1.53105 x x x x x x x 1.71076 x x x 1.57736 x x x x x x
x x x x x x x x x x 1.22265 x 1.46658 x x x x x x x x 1.97835 x 1.4059 x x x x 1.38893 x 1.73928 x x x 1.13301 x x x x x x 1.51266 1.85632 1.06918 1.08912 x x x
x x x x x x x x x x 1.93824 x x 1.15376 x x x x x x x x x 1.37146 1.21006 x x 1.38893 x x x 1.66426 x x x x 1.49737 1.31392 1.36737 1.35118 1.25922 x x x x x x x
x x x x x x x x x x 1.70932 x x x 1.78958 x x x x x 1.61613 x x x x x 1.53105 x x x x x x 1.01974 x x x 1.53939 x x x 1.69952 x x x x x x
x x x x 1.25345 x x x x x x x x x x x 1.90993 1.76886 x x x x x x x x x 1.73928 x x x 1.05697 x x 1.11392 x x x x x x x 1.35905 1.527 1.3608 1.19098 1.22235 x
x x x x 1.08944 1.33248 x x x x x x x x x x 1.20661 x 1.53023 x x x x x 1.39195 x x x 1.66426 x 1.05697 x x x x x x x 1.97238 1.93274 1.76974 x x x x x x 1.05863
x x x x x x x x x x x 1.73451 x x x x 1.2079 x x 1.45569 x x x x x x x x x x x x x 1.16883 x 1.52226 x x x x x x x x x 1.20261 x 1.33539
x x x x x x x x x x x 1.38332 x x 1.31054 x 1.46994 x x x 1.60929 x x x x x x x x 1.01974 x x 1.16883 x x x x x x x x x x x x x 1.74398 x
x x x x x x x x x x x 1.30693 1.78019 x x x x x x x x 1.76921 x x x x x 1.13301 x x 1.11392 x x x x x x x x x x x 1.1288 1.69377 1.12686 1.91775 1.03383 x
x x x x 1.4328 x 1.01789 x x x x x x x x x x x x x x x 1.58527 x x 1.70246 x x x x x x 1.52226 x x x x x x x x x x x x 1.4965 x 1.68411
x x x x x x x x x x x x x x x x x x x x x x x x 1.89186 1.98514 x x 1.49737 x x x x x x x x 1.84162 1.2419 1.63483 x x x x x x x 1.09189
x x x x x x x x x x x x x x x x x x x x x x x x 1.60453 x 1.71076 x 1.31392 1.53939 x x x x x x 1.84162 x x x 1.63326 1.81244 x x x x x x
x x x x x 1.07438 x x x x x x x 1.74855 x x x x x x x x x x 1.47707 x x x 1.36737 x x 1.97238 x x x x 1.2419 x x 1.04034 1.29556 x x x x x x 1.22245
x x x x x x x x x x x x x x x x x x 1.95878 x x x x x 1.76987 x x x 1.35118 x x 1.93274 x x x x 1.63483 x 1.04034 x 1.17181 x x x x x x 1.69892
x x x x x 1.40389 x x x x x x x 1.62743 x x x x x x x x x x 1.14891 x x x 1.25922 x x 1.76974 x x x x x 1.63326 1.29556 1.17181 x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x 1.57736 1.51266 x 1.69952 x x x x x x x 1.81244 x x x x 1.62872 x 1.11441 x 1.99162 x
x x x x x x x x x x x x 1.45884 x x x x x x x x x x x x x x 1.85632 x x 1.35905 x x x 1.1288 x x x x x x 1.62872 x 1.95275 1.1152 x 1.38396 x
x x x x x x x x x x x x x x x x x 1.75345 x x x 1.31586 x x x x x 1.06918 x x 1.527 x x x 1.69377 x x x x x x x 1.95275 x 1.25261 1.95793 x x
x x x x x x x x x x x x x x x x x 1.93443 x x x 1.65585 x x x x x 1.08912 x x 1.3608 x x x 1.12686 x x x x x x 1.11441 1.1152 1.25261 x x 1.69926 x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.19098 x 1.20261 x 1.91775 1.4965 x x x x x x x 1.95793 x x 1.95284 1.21154
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.22235 x x 1.74398 1.03383 x x x x x x 1.99162 1.38396 x 1.69926 1.95284 x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.05863 1.33539 x x 1.68411 1.09189 x 1.22245 1.69892 x x x x x 1.21154 x x
