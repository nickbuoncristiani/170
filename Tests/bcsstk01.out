10 12 35 13 10 12 33 36 7 8 27 38 25 32 17 34 33 36 5 32 19 18 44 45 43 13 3 8 7 36 23 1 2 6 39 29 24 11 9 10
22
10 15
35 35
13 13
33 33
36 36
7 7
8 8
38 38
32 32
17 17
34 34
18 18
44 44
45 45
43 43
36 46
23 23
1 16
2 2
39 39
24 4 24
11 11 28
9 9
